INSTALL.md

```diff
@@ -4,6 +4,7 @@ These are instructions for building stellar-core from source.
 
 For a potentially quicker set up, the following projects could be good alternatives:
 
+* stellar-core on [Gitpod](https://gitpod.io/#https://github.com/stellar/stellar-core)
```

Comment:
It would make sense to qualify that, yes, and maybe remove it outright.  But my idea in putting that there was not that anybody try to run a stellar-core that they _use_ from Gitpod, but quite possibly that they run one that they experiment on, analyzing live, to learn how it works -- the same reason that I wanted to make Tracy work, as described in [my comment above](https://github.com/stellar/stellar-core/pull/2596/files#r446574112).